saturnbase-gpu/install-miniconda.bash

```diff
@@ -4,7 +4,7 @@ set -ex
 
 cd $(dirname $0)
 
-MINICONDA_VERSION=py37_4.8.2
+MINICONDA_VERSION=py37_4.9.3
```

Comment:
I think you meant 4.9.2

```suggestion
MINICONDA_VERSION=py37_4.9.2
```

There isn't a `py37_4.9.3`: https://repo.anaconda.com/miniconda/

Reply:
Right you are - fixed in `194d122`.